tests/test_log.py

```diff
@@ -34,17 +33,19 @@ def test_log(self, mocker: Any) -> None:
         self.repo_sandbox.commit()
         childs_second_commit_hash = get_current_commit_hash()
 
-        log_content = launch_command('log')
+        log_content_no_branch_name = launch_command('log')
+        log_content_short_branch_name = launch_command('log', 'child')
+        log_content_full_branch_name = launch_command('log', 'refs/heads/child')
 
-        assert childs_first_commit_hash in log_content, \
+        assert childs_first_commit_hash in log_content_short_branch_name and log_content_no_branch_name and log_content_full_branch_name, \
```

Comment:
Huh how is it executed? 🤔 does `X in A and B and C` really check `X's presence in all of `A`, `B`,`C`? 🤔 Isn't it interpreted just as `(X in A) and B and C`, which would be somewhat useless here?